jsconfig.json

```diff
@@ -8,7 +8,7 @@
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"sourceMap": true,
-		"strict": true,
+		"strict": false,
```

Comment:
how come you're turning off strict mode?

Reply:
With strict mode off, we still get this benefit.  I think it's because of `"checkJs": true,`

![image](https://github.com/user-attachments/assets/19f2b46b-8e05-4b5f-8bd5-421d91a9a80c)

With strict mode *on\*, we get this sort of thing:

![image](https://github.com/user-attachments/assets/725e5855-38d7-4763-87d8-4ec16b4ff45d)

It's complaining that the variable implicitly has a type of null, but I am reassigning it to a different type when setting it to the found campaign object.

It can be fixed like this:

![image](https://github.com/user-attachments/assets/5c4e87f9-34b5-443b-969d-da4d693a7b1d)

But ¯\_(ツ)_/¯ I think my leaning would be to embrace the chaos of JS, I've a craving for some wild west style development.

Reply:
That being said, it's not a non-negotiable by any means.  I'm even open to using TS.